tests/E2E/test/campaigns/full/15_design/3_link_widget/01_linkWidget.js

```diff
@@ -285,6 +279,7 @@ scenario('Create, edit, delete LinkWidget with different HOOK ', () => {
       test('should change the Front Office language to "English"', () => client.changeLanguage());
       test('should check in the FO if the block is displayed', () => client.waitForVisible(AccessPageFO.nav_full_width_link_widget.replace('%NAVFULLWIDTHLINKWIDGET', 'displayNavFullWidth' + " " + date_time)));
       test('should go to the back office the page', () => client.switchWindow(0));
+      test('should to the table', () => client.scrollTo(LinkWidget.last_widget_drag_in_displayFooter_block.replace('%HOOK', 'displayNavFullWidth')));
```

Comment:
`should go to the table` ?

Reply:
good call thanks

Reply:
or `should scroll to the table`

Reply:
already done :+1:

Reply:
There's still several occurrences of `should to the table` to be updated